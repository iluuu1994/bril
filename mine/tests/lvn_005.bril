@main(a: int, b: int) {
  c: int = add a b;
  print c;
  d: int = add b a;
  print d;
}
