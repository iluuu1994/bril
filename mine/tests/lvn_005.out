@main(a: int, b: int) {
  c: int = add a b;
  print c;
  print c;
}
